CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/utilities/MarkdownTree.java

```diff
@@ -340,11 +363,11 @@ private void parseString(final MarkdownNode currentNode, String text) {
                         // and the next line does not start with the same amount of tabs or it does start with the same amount of tabs
                         // and there is a new tab right after that OR there are no tabs on the list item and the next line contains a tab
                         // then change end index of the \n for this next line
-                        while (endIndex != -1 && ((!tabString.isEmpty()
-                                && (text.indexOf(tabString, endIndex + 1) != endIndex + 1
-                                || (text.indexOf(tabString, endIndex + 1) == endIndex + 1
+                        while (endIndex != -1 && ((!tabString.toString().isEmpty()
```

Comment:
worth noting that StringBuilder I think has an empty string check so you don't need to convert to string first there.
Also given you're not adding to `tabString` once you get to this point, it might be worth just converting to string in a separate variable beforehand and just using that rather than calling `toString` numerous times